UndertaleModTool/Editors/UndertaleRoomEditor.xaml.cs

```diff
@@ -1246,13 +1254,15 @@ private void AddLayer<T>(LayerType type, string name) where T : Layer.LayerData,
                 }
             }
 
+            // "layerDepth" is "long", because otherwise it couldn't be checked if incremented value is larger than "Int32.MaxValue",
+            // because then it will overflow.
```

Comment:
```suggestion
            // "layerDepth" is "long", because otherwise one can't check if the incremented value is larger than "Int32.MaxValue",
            // because then it'd overflow.
```

Reply:
Yeah. One is a generic term for a person.